packages/plugin-core/src/services/NoteSyncService.ts

```diff
@@ -28,91 +25,72 @@ import { Logger } from "../logger";
 import { VSCodeUtils } from "../vsCodeUtils";
 
 /**
- * Interface for a service that synchronizes notes from the client to the engine
+ * Interface for a service that processes text document changes from vscode. TODO: Rename filename
```

Comment:
Will renaming it now make it harder for you to see the diffs on github? I was thinking of doing it at the end or in separate pr

Reply:
O - for the sake of review not a problem (I think the diffs should also be aware of the rename).  Mostly just wanted to add a button to 'resolve conversation' to make sure you don't forget to do prior to merging, since I've forgotten to do stuff like that before haha.